apis/v1alpha1/generated.pb.go

```diff
@@ -1,11639 +0,0 @@
-/*
```

Comment:
While this file is removed, it will be present in local branches of existing PRs and could be added by mistake by contributors.
Reviewers can ensure that this doesn't happen but we can automate this by failing the verify job if these deleted files are present. Not a blocker for this PR to get in.